atlp-rwanda/script-heroes-frontend
# #174797463 add signup page

#### What does this PR do?
Adding the signup page to the web
#### Description of Task to be completed?
- Creating a Signup form
- Validating users input before submitting
- Routing signup from the landing page
#### How should this be manually tested?
- Clone the repo `git clone https://github.com/atlp-rwanda/script-heroes-frontend.git`
- Switch your working directory `cd script-heroes-frontend`
- checkout the branch `git checkout ft-user-signup-#174797463`
- Create a `.env` file and update it with data from `.env-sample`
- Install all dependencies `yarn install`
- Start the server `yarn run dev`
- Click on the signup link found in the navigation bar
- Fill in your data in the Signup form found on the page
> You should see an error if you inserted the wrong data
> You should see a successful message if your data accepted
#### What are the relevant pivotal tracker stories?
[#174797463](https://www.pivotaltracker.com/story/show/174797463)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/44426710/99188073-c3315300-2762-11eb-9f31-9955e5883733.png)
